Technical-Steering-Committee/principles.md

```diff
@@ -0,0 +1,43 @@
+# Principles
+
+Zowe is a project that aims to bring modern interfaces - API, command line, and visual – to z/OS via an open source community. In case of disagreement within the Zowe community, the principles outlined here need to be taken into account when mediating.
+
+## Zowe simplifies the life of Modern Mainframers
+**Principle:** Modern and effective interfaces
+
+We will provide the foundations and tools for all the vendors existing within the ecosystem to easily interface to and develop more interfaces to z/OS. The users should be able to leverage the provided tools and interfaces to get more value out of their mainframe environments. Examples domains for such tools and interfaces include web, mobile, APIs (C interfaces to HLASM, other programming interfaces, REST APIs, ...) and modern IDEs.
+
+## Zowe does not compromise on z/OS security standards
+**Principle:** Security is paramount
+
+We commit to secure development practices, source code scans and vulnerability testing. We will never introduce a feature or change that circumvents or allows circumvention of z/OS security infrastructure. If vulnerabilities are ever found they will be flagged with fix or workaround provided.
+
+## Transparent and open implementation, processes and deliveries
+**Principle:** Open first
+
+We will plan and develop transparently and in open. We will build an inclusive environment where anyone willing can easily find a way to contribute. 
+
+## Collaboration between z/OS and other platforms - both mainframe and distributed - benefits both sides
+**Principle:** Hybrid
+
+We will bring lessons from the non-z/OS world to the z/OS platform as well as providing the opportunity for the non-z/OS world to benefit from the knowledge gained in the z/OS world in order to build secure and performant hybrid environments in which z/OS plays core role. 
+
+## Decision made within the project are based on community participation and supported by data
+** Principle:** Agile and data driven
+
+We believe in agile development practives with small teams, iterative development, etc. and we are data-driven with respect to quality and performance metrics.
```

Comment:
This here is maybe missing information on the decision making processes. What decisions (high level) the TSC would be responsible for, e.g. defining Zowe's roadmap, and how prioritisation would work, e.g. how is the community feedback used for it.

Reply:
Just to explain a bit on the level of the document. The TSC and ZAC changes shouldn't impact the document. It needs to be quite general. That's one of the reasons why details like specific processes and specific groups responsible for them aren't mentioned here. 

Some of the other high level responsibilities are mentioned in the TSC charter or are directly applicable from OMP or Linux Foundation charter. 

Just FYI. TSC is responsible for any decisions with material impact to the Zowe unless it already delegated it to either ZAC, squads or working groups. The other topics are relevant and interesting but not really sure how